package.json

```diff
@@ -45,36 +45,40 @@
     "url": "git+https://github.com/webex/react-ciscospark.git"
   },
   "contributors": [
-    "Adam Weeks <adweeks@cisco.com>",
-    "Bernie Zang <nzang@cisco.com>",
+    "Arash Koushkebaghi <akoushke@cisco.com>",
     "Moriah Maney <momaney@cisco.com>",
-    "Arash Koushkebaghi <akoushke@cisco.com>"
+    "Matt Norris <matnorri@cisco.com> (http://mattnorris.me)",
+    "Adam Weeks <adweeks@cisco.com>",
+    "Bernie Zang <nzang@cisco.com>"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/webex/react-ciscospark/issues"
   },
   "homepage": "https://github.com/webex/react-ciscospark#readme",
   "dependencies": {
-    "@ciscospark/common": "1.43.2",
-    "@ciscospark/helper-html": "1.43.2",
-    "@ciscospark/internal-plugin-conversation": "1.47.3",
-    "@ciscospark/internal-plugin-encryption": "1.47.3",
-    "@ciscospark/internal-plugin-feature": "1.47.3",
-    "@ciscospark/internal-plugin-flag": "1.47.3",
-    "@ciscospark/internal-plugin-mercury": "1.47.3",
-    "@ciscospark/internal-plugin-metrics": "1.47.3",
-    "@ciscospark/internal-plugin-presence": "1.47.3",
-    "@ciscospark/internal-plugin-search": "1.47.3",
-    "@ciscospark/internal-plugin-team": "1.47.3",
-    "@ciscospark/internal-plugin-user": "1.47.3",
-    "@ciscospark/plugin-authorization": "1.47.3",
-    "@ciscospark/plugin-logger": "1.47.3",
-    "@ciscospark/plugin-people": "1.47.3",
-    "@ciscospark/plugin-phone": "1.47.3",
-    "@ciscospark/spark-core": "1.47.3",
+    "@ciscospark/common": "1.49.3",
```

Comment:
@adamweeks should I bump this to the latest?

Reply:
sure!